.github/workflows/ci-test.yml

```diff
@@ -86,6 +86,7 @@ jobs:
 
   unit-test:
     runs-on: ubuntu-latest-xl
+    timeout-minutes: 70
```

Comment:
Could you add a comment explaining how 70 was chosen? For future ours' sake :)

Reply:
It was dropped to 40, 70 was way overkill, should 32-33 at most.